src/main/java/bio/terra/common/logging/LoggingUtils.java

```diff
@@ -27,6 +27,25 @@
 
   private LoggingUtils() {}
 
+  // A simple string which can be included as a key in JSON logging output. This is intended to
+  // trigger log-based alerting to notify developers of unexpected errors.
+  public static final String ALERT_KEY = "terraLogBasedAlert";
+
+  public static Map<String, Boolean> alertObject() {
+    return Collections.singletonMap(ALERT_KEY, true);
+  }
+
+  /**
+   * Wrapper around logger.error() which also includes the alertObject defined above.
+   *
+   * <p>This allows callers to log at ERROR level and also trigger any environment-appropriate
+   * logging set up wherever their code is running. For logs which also need to include objects,
+   * callers can just include the alertObject() as an additional structured object to log.
+   */
+  public static void logAlert(Logger logger, String message) {
+    logger.error(message, alertObject());
```

Comment:
You could log at SEVERE level here I think. I so rarely get to do that.

Reply:
Unfortunately that's a `java.util.Logging` level, not SLF4J :(